packages/opentelemetry-node/docs/supported-technologies.mdx

```diff
@@ -15,9 +15,15 @@ Elastic stack, so it is strongly recommended to be using a recent 8.x version.
 
 ## Node.js versions
 
-The Elastic Distribution for OpenTelemety Node.js supports Node.js v14 and later.
+The Elastic Distribution for OpenTelemety Node.js supports Node.js v14.18.0 and later.
 This follows from the [OpenTelemetry JS supported runtimes](https://github.com/open-telemetry/opentelemetry-js#supported-runtimes).
 
+<!--
+Dev Notes on supported Node.js versions:
+- v14.18.0 or later is required for `node:`-prefixed core module loading, as
+  used by gaxios@6.7.0, a transitive dep of `@opentelemetry/resource-detector-gcp`.
+-->
```

Comment:
I *think* we can have HTML comments in our .mdx files? @bmorelli25 ?

Otherwise, I guess we could find another place to document this, e.g. in the test.yml file.

Reply:
You'll be able to do this in the asciidoc files when you rebase this PR

Reply:
cc @colleenmcginnis in case this PR gets merged before the MDX ➡️ Asciidoc PR.